manifests/supervisorcluster/1.21/cns-csi.yaml

```diff
@@ -506,6 +506,11 @@ webhooks:
         operations:  ["CREATE", "UPDATE", "DELETE"]
         resources:   ["persistentvolumeclaims"]
         scope: "Namespaced"
+      - apiGroups:   ["snapshot.storage.k8s.io"]
+        apiVersions: ["v1"]
+        operations:  ["CREATE", "UPDATE", "DELETE"]
```

Comment:
why webhook needs Create, update and Delete permissions on snapshot?

Reply:
ignore this comment. @deepakkinni confirmed it is for intercepting requests for create, update and delete in the webhook.